spoon/form/form.php

```diff
@@ -1096,7 +1099,12 @@ public function validate()
 		if($this->getUseToken())
 		{
 			// token not available?
-			if(!SpoonSession::exists('form_token')) $errors .= $this->tokenError;
+			if (!session_id()) {
+				@session_start();
```

Comment:
Hmm, if no session exists it means there is no token, so therefor this will always result in an error?
I don't like the `session_start` because it prevents caching, it sets cookies, ...